addons/stock_delivery/wizard/choose_delivery_package.py

```diff
@@ -24,7 +24,8 @@ def _compute_weight_uom_name(self):
     def _compute_shipping_weight(self):
         for rec in self:
             move_line_ids = rec.picking_id.move_line_ids.filtered(lambda m:
-                float_compare(m.quantity, 0.0, precision_rounding=m.product_uom_id.rounding) > 0
+                m.picked
```

Comment:
```suggestion
                m.picked or not m.move_id.picked
```
The initial solution works well within the barcode app, but if you try to do the same thing through the detailed operations without trying to tick the "picked" checkbox, then it wouldn't work anymore while it was up until now.

Reply:
I understand better how `picked` is used now, thank you!